src/NzbDrone.Core/Datastore/Migration/131_indexer_priority.cs

```diff
@@ -0,0 +1,82 @@
+using FluentMigrator;
+using Newtonsoft.Json.Linq;
+using NzbDrone.Core.Datastore.Migration.Framework;
+using System;
+using System.Collections.Generic;
+using System.Data;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(131)]
+    public class indexer_priority : NzbDroneMigrationBase
```

Comment:
Do we need to do a migration? If a value isn't set then the default should apply. I think if we ever wanted to change the default we might want to update the DB for existing records, but it should be fine for now.

Reply:
I prefer to write a migration instead of implementing a fallback to default value if the value is missing because it's more maintainable IMO. Knowing that the Priority field will always be available simplifies code and obseletes fallback code. Again my opinion so if you disagree I can remove the migration and test.

Reply:
You don't need to implement anything, if nothing is set in the settings JSON in the DB it'll use the default of the model.

Reply:
Ok, I'll remove the migration